porch/pkg/git/git.go

```diff
@@ -784,6 +784,8 @@ func (r *gitRepository) pushAndCleanup(ctx context.Context, ph *pushRefSpecBuild
 		RefSpecs:          specs,
 		Auth:              auth,
 		RequireRemoteRefs: require,
+		// TODO(justinsb): Need to ensure this is a compare-and-swap
+		Force: true,
```

Comment:
What is this for? I don't see it used anywhere

Reply:
Ah, it's because when we update a package now it isn't necessarily a fast-forward, because we're effectively doing a `git rebase`, which needs a force when we `git push`.  We can do better than `git push -f` because we want to detect concurrent activities, but we need to track and pass the current expected version on git.